src/main/java/com/assignment1/userinterface/UserInterfaceHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.assignment1.userinterface;
+
+import com.assignment1.exceptions.RuntimeExceptionCustom;
+import com.assignment1.item.ItemEntity;
+import java.util.List;
+
+/** Interface to declare methods. */
+public interface UserInterfaceHandler {
+
+  /** Method to take input. */
+  ItemEntity takeInput() throws RuntimeExceptionCustom;
+
+  /** Method to show message. */
+  boolean showUsageMessage();
+
+  /** Method to give output. */
+  boolean giveOutput(List<ItemEntity> itemArray);
+}
```

Comment:
as i mentioned last time this interface make no sense. think about it, it has methods that perform read/write operations on an Item, where does a `User` come into picture?

Maybe an interface like `ItemReaderWriter` is better suited or 2 separate interfaces can be defined `ItemReader` and `ItemWriter` with methods `read()` and `write()`

Reply:
Avoided the interface and implemented the read() functionality as method of ItemEntity class. Now, reading repends on properties of class. So, it makes more sense to have read() as method. Avoided any king of Interface now for this reading and writing behaviour.